pallets/parachain-staking/src/migrations.rs

```diff
@@ -15,131 +15,236 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
-
-use crate::{
-	inflation::InflationInfo,
-	pallet::*,
-	types::{BalanceOf, Releases},
-};
+#[cfg(feature = "try-runtime")]
+use frame_support::ensure;
 use frame_support::{dispatch::Weight, traits::Get};
-use kilt_primitives::constants::MAX_COLLATOR_STAKE;
-use sp_runtime::traits::Zero;
-
-pub mod v2 {
+use sp_runtime::{
+	codec::{Decode, Encode},
+	traits::Zero,
+};
+use sp_std::marker::PhantomData;
+
+use crate::*;
+
+mod v2;
```

Comment:
Note: Moved all code to separate files. So you don't need to review `migration/v{1,2,3,4}.rs`. Neither have any changes have been made nor did I add a new migration. Since we have already enforced the bounds before, we are safe to upgrade without migration. The template matches the one from #226.